roofit/roofitcore/src/RooGradMinimizerFcn.cxx

```diff
@@ -0,0 +1,238 @@
+/*****************************************************************************
+ * Project: RooFit                                                           *
+ * Package: RooFitCore                                                       *
+ * @(#)root/roofitcore:$Id$
+ * Authors:                                                                  *
+ *   AL, Alfio Lazzaro,   INFN Milan,        alfio.lazzaro@mi.infn.it        *
+ *   PB, Patrick Bos,     NL eScience Center, p.bos@esciencecenter.nl        *
+ *   VC, Vince Croft,     DIANA / NYU,        vincent.croft@cern.ch          *
+ *                                                                           *
+ *                                                                           *
+ * Redistribution and use in source and binary forms,                        *
+ * with or without modification, are permitted according to the terms        *
+ * listed in LICENSE (http://roofit.sourceforge.net/license.txt)             *
+ *****************************************************************************/
+
+//////////////////////////////////////////////////////////////////////////////
+//
+// RooGradMinimizerFcn is am interface class to the ROOT::Math function
+// for minimization. See RooGradMinimizer.cxx for more information.
+//
+
+#include "RooGradMinimizerFcn.h"
+
+#include "RooAbsArg.h"
+#include "RooAbsPdf.h"
+#include "RooArgSet.h"
+#include "RooRealVar.h"
+#include "RooMsgService.h"
+#include "RooMinimizer.h"
+
+#include "Riostream.h"
+#include "TIterator.h"
+#include "TClass.h"
+#include "Fit/Fitter.h"
+#include "Math/Minimizer.h"
+
+#include <algorithm> // std::equal
+#include <iostream>
+
+RooGradMinimizerFcn::RooGradMinimizerFcn(RooAbsReal *funct, RooMinimizer *context, bool verbose)
+   : RooAbsMinimizerFcn(RooArgList( * std::unique_ptr<RooArgSet>(funct->getParameters({})) ), context, verbose),
```

Comment:
```suggestion
   : RooAbsMinimizerFcn(RooArgList( * std::unique_ptr<RooArgSet>(funct->getParameters(RooArgSet{})) ), context, verbose),
```
If you just use `{}`, I get an ambiguous function call error when I compile. You'd need to construct a RooArgSet explicitly.